lib/logstash/pluginmanager/install.rb

```diff
@@ -1,82 +1,128 @@
 require 'clamp'
 require 'logstash/namespace'
 require 'logstash/environment'
-require 'logstash/pluginmanager'
 require 'logstash/pluginmanager/util'
-require 'rubygems/dependency_installer'
-require 'rubygems/uninstaller'
 require 'jar-dependencies'
 require 'jar_install_post_install_hook'
 require 'file-dependencies/gem'
 
+require "logstash/gemfile"
+require "logstash/bundler"
+
 class LogStash::PluginManager::Install < Clamp::Command
+  parameter "[PLUGIN] ...", "plugin name(s) or file"
```

Comment:
I think 
`bin/plugin install-development-dependencies` 
is good  and 
`bin/plugin install --no-verify` 
is ok to launch a bare "bundle install". 
or maybe
`bin/plugin install --bare` ?